docs/pages/versions/unversioned/sdk/firebase-analytics.mdx

```diff
@@ -10,6 +10,8 @@ import PlatformsSection from '~/components/plugins/PlatformsSection';
 import { Terminal } from '~/ui/components/Snippet';
 import { InlineCode } from '~/components/base/code';
 
+> **warning** **Deprecated.** This module will be removed in SDK 48. Check out our [Migration Guide](https://expo.fyi/rnfs-migration-guide) for help migrating to [React Native Firebase](https://rnfirebase.io/).
```

Comment:
```suggestion
> **warning** **Deprecated:** This module will be removed in SDK 48. Check out our [Migration Guide](https://expo.fyi/rnfs-migration-guide) for help migrating to [React Native Firebase](https://rnfirebase.io/).
```

I'd suggest using `:` instead of `.` after the word Deprecated. It's not exactly a convention, just a small nit because of the pattern we follow across all docs when putting up a deprecation notice 😄